client/client.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+	pb "github.com/BetterGR/course-microservice/course_protobuf"
+)
+
+const (
+	// serverAddress defines the address of the gRPC server
+	serverAddress = "localhost:50052"
+)
+
+func main() {
+	logger, err := zap.NewProduction()
+	if err != nil {
+		log.Fatalf("Failed to initialize logger: %v", err)
+	}
+	defer logger.Sync()
+
+	conn, err := grpc.Dial(serverAddress, grpc.WithInsecure())
+	if err != nil {
+		logger.Fatal("Failed to connect to server", zap.Error(err))
+	}
+	defer conn.Close()
+
+	client := pb.NewCourseServiceClient(conn)
+
+	// Call example RPC methods
+	createCourse(client, logger)
+	getCourse(client, logger)
+}
+
+func createCourse(client pb.CourseServiceClient, logger *zap.Logger) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+
+	req := &pb.CreateCourseRequest{
+		Name:        "Algorithms 1",
```

Comment:
Avoid spaces in data instances. Think of such data as enums that are not for user-display.